internal/api/shared/migrate/testdata/old_grafana_panels_perses_dashboard.json

```diff
@@ -47,7 +47,8 @@
             "spec": {
               "text": "<p>This dashboard provides information on OTF Front-End queues.<br/>Minimum OTF FE version required: 18.0.0.68 / 19.0.0.29<br/>\nMore information on FE metrics used here: <a href=\"https://rndwww.nce.amadeus.net/confluence/display/OTFTeam/OTF+Monitoring+-+FE+metrics\">Confluence link</a>\n</p>"
             }
-          }
+          },
+          "queries": []
```

Comment:
mmm this should not appear if it is empty. But I guess it's coming from the other branch. I will look there

Reply:
Hmm if you really want it to not generate an empty array when there's no query, there's most probably a new check to add in the cue logic here https://github.com/perses/perses/pull/1072/files#diff-d4979002465bf1e3768d854a8ba3c0caa4802753bf17e356b3c3a6015b3460ea

Reply:
I updated my branch to add `omitempty` for `queries`. Can you please pull the new changes?